tests/17-tun-rpl-br/02-border-router-cooja-tsch.csc

```diff
@@ -52,7 +52,7 @@ make -j border-router.cooja TARGET=cooja MAKE_MAC=MAKE_MAC_TSCH</commands>
       <description>Cooja Mote Type #2</description>
       <source>[CONTIKI_DIR]/examples/hello-world/hello-world.c</source>
       <commands>make TARGET=cooja clean
-make -j hello-world.cooja TARGET=cooja MAKE_MAC=MAKE_MAC_TSCH</commands>
+make -j hello-world.cooja TARGET=cooja MAKE_MAC=MAKE_MAC_TSCH DEFINES=TSCH_CONF_DEFAULT_HOPPING_SEQUENCE=TSCH_HOPPING_SEQUENCE_1_1</commands>
```

Comment:
so, no more coverage of channel hopping in cooja?

Reply:
As you know if there are multiple channels, the joining time becomes potentially unbounded. In practice, this means that the test timeout should be increased, especially if the 4 channels default is used. IMO it's better to avoid doing it for this test as its a real-time test (speedlimit=100%) and instead leave channel hopping testing for other tests, which can be executed much faster. However, if you want, I can increase the test timeout (from the current 120 seconds) and retain the channel hopping.

Reply:
Do we have tests with channel hopping on speedlimit > 100%?

Reply:
Sure, `tests/07-simulation-base` has a number of TSCH tests with the default channel hopping sequence and unlimited speed, if that's what you mean.

Reply:
Then it's good with me.